std-bits/table/src/main/java/org/enso/table/excel/ExcelRow.java

```diff
@@ -34,10 +37,17 @@ public Object getCellValue(int column) {
     CellType cellType = getCellType(cell);
     switch (cellType) {
       case NUMERIC:
-        if (DateUtil.isCellDateFormatted(cell)) {
-          return cell.getLocalDateTimeCellValue().toLocalDate();
+        double dblValue = cell.getNumericCellValue();
+        if (DateUtil.isCellDateFormatted(cell) && DateUtil.isValidExcelDate(dblValue)) {
+          var dateTime = DateUtil.getLocalDateTime(dblValue);
+          if (dateTime.isBefore(LocalDateTime.of(1900, 1, 2, 0, 0))) {
+            return dateTime.toLocalTime();
+          }
```

Comment:
I don't understand this logic. What is the siginficance of this date? Can we give it a name and some short comment explaining why a date-time before 1900 is treated as time-of-day?